docs/network-mobile/mobile-proof-of-coverage.mdx

```diff
@@ -273,6 +332,79 @@ adding more features and data for MOBILE Rewards to the IOT Network Explorer bec
 impossible as the MOBILE Network grows. The first iteration of the MOBILE Explorer proposes a basic
 map overlay of 5G data coverage with signal strength.
 
+---
+
+### Service Provider Hex Boosting
+Any Service Provider in the MOBILE subDAO can boost hexes to increase PoC rewards for certain hexes.
```

Comment:
Let's break this out as its own page. Happy to help get it stubbed-out

Reply:
Works for me. Are you able to create a template page for me and I can fill it in?

Reply:
Also, can you provide the link to that page in here?